docs/finite-states.mdx

```diff
@@ -164,7 +164,7 @@ Read more about [tags](tags.mdx).
 
 ## Meta
 
-Meta data is static data that describes relevant properties of a state node. You can specify meta data on the `.meta` property of any state node. This can be useful for displaying information about a state node in a UI, or for generating documentation.
+Meta data is static data that describes relevant properties of a state node. You can specify meta data on the `.meta` property of any state node. This can be useful for displaying information about a state node in a UI, or for generating documentation. You may access the state node's `meta` properties at runtime with the following: `yourActor.getSnapshot().getMeta()`. Note only "top level" `meta` and the current state's `meta` fields will be returned in `getMeta()`.
```

Comment:
```suggestion
Meta data is static data that describes relevant properties of a state node. You can specify meta data on the `.meta` property of any state node. This can be useful for displaying information about a state node in a UI, or for generating documentation. You may access the state node's `meta` properties at runtime with the following: `yourActor.getSnapshot().getMeta()`. Note that only the `meta` data for the active state nodes (including the root state node) will be returned in `getMeta()`.
```